### PR TITLE
[tw] Localize Chaos Engineering for Traditional Chinese (#2412)

### LINKED ISSUE #2412: [tw] Localize `Chaos Engineering` for Traditional Chinese
1. Localize `Chaos Engineering` for Traditional Chinese
2. https://glossary.cncf.io/chaos-engineering/

### PR DESCRIPTION
**Pull request by Fionn88**
### Describe your changes

Origin content from https://glossary.cncf.io/chaos-engineering/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2412 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+## 是什麼 {#what-it-is}
+
+混沌工程或 CE，是在正式環境中對[分散式系統](/zh-tw/distributed-system/)進行實驗的專業，以建立對系統在承受混亂和意外情況下時能力的信心。
+
+## 解決的問題 {#problem-it-addresses}
+
+[SRE](/zh-tw/site-reliability-engineering/) 和 [DevOps](/zh-tw/devops/) 實踐注重提高產品的彈性和[可靠性](/zh-tw/reliability/)技術。
+系統在故障容錯時確保有足夠服務品質的能力通常是軟體開發的要求。有幾個方面可能導致應用程式發生故障，例如基礎設施、平台或(基於[微服務](/zh-tw/microservices/))的應用程式的其他部分。
```
Comment (johnlinp) [inline content/zh-tw/chaos-engineering.md:15]: 通常圓括號內的字在拿掉時也不會影響到文章的閱讀，但你這個地方在圓括號內的字拿掉時會怪怪的。

我建議改成：
```suggestion
系統在故障容錯時確保有足夠服務品質的能力通常是軟體開發的要求。有幾個方面可能導致應用程式發生故障，例如基礎設施、平台或(基於[微服務](/zh-tw/microservices/)的)應用程式的其他部分。
```

File: content/zh-tw/chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+在正式環境中高頻率部署新功能會增加導致停機和嚴重事件發生的可能性，這對業務會產生重大影響。
+
+## 如何幫助我們 {#how-it-helps}
+
+混沌工程是一種滿足彈性需求的技術。它用於實現對基礎設施、平台和應用程序故障容錯。
```
Comment (johnlinp) [inline content/zh-tw/chaos-engineering.md:20]:
```suggestion
混沌工程是一種滿足彈性需求的技術。它用於達成對基礎設施、平台和應用程序故障容錯。
```

原文：

> Chaos engineering is a technique to meet resilience requirements. It is used to achieve resilience against infrastructure, platform, and application failures.

File: content/zh-tw/chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+混沌工程師使用混沌實驗來主動注入隨機故障，以驗證應用程序、基礎設施或平台是否能夠自我修復，並且故障不會明顯影響客戶。
+混沌實驗旨在發現盲點（例如監控或自動擴展技術），並在嚴重事件期間提升團隊之間的溝通。
+這種方法有助於提高系統的彈性和團隊對複雜系統的信心，尤其是生產環境。
```
Comment (johnlinp) [inline content/zh-tw/chaos-engineering.md:23]:
```suggestion
這種方法有助於提高系統的彈性和團隊對複雜系統的信心，尤其是正式環境。
```

**Review by hwchiu — APPROVED** [review]
LGTM, 感謝

**Comment by johnlinp** [comment]
不好意思，我發現你這個 pr 有 Site Reliability Engineering 跟 Chaos Engineering 兩個 commits ，可以請你把 Site Reliability Engineering 的 commit 拿掉嗎？謝謝。

**Comment by Fionn88** [comment]
> 不好意思，我發現你這個 pr 有 Site Reliability Engineering 跟 Chaos Engineering 兩個 commits ，可以請你把 Site Reliability Engineering 的 commit 拿掉嗎？謝謝。

想跟您確認一下是使用rebase的方式，將兩個 Git commit 合併成一個是這樣嗎？

**Comment by johnlinp** [comment]
喔我不是想把這兩個 commits 合併
我想的是把 Site Reliability Engineering 的 commit 拿掉

希望最後的成果是
這個 pr 單純只看得到 Chaos Engineering 的內容
不會有 Site Reliability Engineering 的內容
謝謝

**Comment by Fionn88** [comment]
> 喔我不是想把這兩個 commits 合併 我想的是把 Site Reliability Engineering 的 commit 拿掉
> 
> 希望最後的成果是 這個 pr 單純只看得到 Chaos Engineering 的內容 不會有 Site Reliability Engineering 的內容 謝謝

我已經修改好了，還麻煩您幫我看一下，這樣是否正確～

**Review by johnlinp — APPROVED** [review]
OK了 謝謝!

File: content/zh-tw/chaos-engineering.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+tags: ["方法論", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+混沌工程或 CE，是在正式環境中對[分散式系統](/zh-tw/distributed-system/)進行實驗的專業，以建立對系統在承受混亂和意外情況下時能力的信心。
+
+## 解決的問題 {#problem-it-addresses}
+
+[SRE](/zh-tw/site-reliability-engineering/) 和 [DevOps](/zh-tw/devops/) 實踐注重提高產品的彈性和[可靠性](/zh-tw/reliability/)技術。
```
Comment (pichuang) [inline content/zh-tw/chaos-engineering.md:14]:
```suggestion
[SRE](/zh-tw/site-reliability-engineering/) 和 [DevOps](/zh-tw/devops/) 實踐注重提高產品的韌性和[可靠性](/zh-tw/reliability/)技術。
```

File: content/zh-tw/chaos-engineering.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+系統在故障容錯時確保有足夠服務品質的能力通常是軟體開發的要求。有幾個方面可能導致應用程式發生故障，例如基礎設施、平台或(基於[微服務](/zh-tw/microservices/)的)應用程式的其他部分。
+在正式環境中高頻率部署新功能會增加導致停機和嚴重事件發生的可能性，這對業務會產生重大影響。
+
+## 如何幫助我們 {#how-it-helps}
+
+混沌工程是一種滿足彈性需求的技術。它用於達成對基礎設施、平台和應用程序故障容錯。
+混沌工程師使用混沌實驗來主動注入隨機故障，以驗證應用程序、基礎設施或平台是否能夠自我修復，並且故障不會明顯影響客戶。
+混沌實驗旨在發現盲點（例如監控或自動擴展技術），並在嚴重事件期間提升團隊之間的溝通。
+這種方法有助於提高系統的彈性和團隊對複雜系統的信心，尤其是正式環境。
```
Comment (pichuang) [inline content/zh-tw/chaos-engineering.md:23]:
```suggestion
這種方法有助於提高系統的韌性和團隊對複雜系統的信心，尤其是正式環境。
```

台灣似乎比較偏好韌性

File: content/zh-tw/chaos-engineering.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+[SRE](/zh-tw/site-reliability-engineering/) 和 [DevOps](/zh-tw/devops/) 實踐注重提高產品的彈性和[可靠性](/zh-tw/reliability/)技術。
+系統在故障容錯時確保有足夠服務品質的能力通常是軟體開發的要求。有幾個方面可能導致應用程式發生故障，例如基礎設施、平台或(基於[微服務](/zh-tw/microservices/)的)應用程式的其他部分。
+在正式環境中高頻率部署新功能會增加導致停機和嚴重事件發生的可能性，這對業務會產生重大影響。
+
+## 如何幫助我們 {#how-it-helps}
+
+混沌工程是一種滿足彈性需求的技術。它用於達成對基礎設施、平台和應用程序故障容錯。
```
Comment (pichuang) [inline content/zh-tw/chaos-engineering.md:20]:
```suggestion
混沌工程是一種滿足韌性需求的技術。它用於達成對基礎設施、平台和應用程序故障容錯。
```

**Review by pichuang — COMMENTED** [review]
Suggest change before approve

**Review by pichuang — APPROVED** [review]
LGTM

**Pull request merged**